src/oxygen/robot3_interface.py

```diff
@@ -0,0 +1,378 @@
+from datetime import datetime
+from datetime import timedelta
+from datetime import timezone
+from robot.result.model import (Keyword as RobotResultKeyword,
+                                Message as RobotResultMessage,
+                                TestCase as RobotResultTest,
+                                TestSuite as RobotResultSuite)
+
+from robot.running.model import TestSuite as RobotRunningSuite
+
+
+class RobotResultInterface(object):
+    SETUP_TYPE = RobotResultKeyword.SETUP_TYPE
+    KEYWORD_TYPE = RobotResultKeyword.KEYWORD_TYPE
+    TEARDOWN_TYPE = RobotResultKeyword.TEARDOWN_TYPE
```

Comment:
Why are we changing the old RF3 interface in any way? To my mind, it should not need to change at all.

Reply:
As far as I can see, these are needed only for tests. I do not quite see why we would change the interface to appease tests; rather, the tests should be re-thought to verify correct things.